packages/nextjs/app/debug/_components/contract/Tuple.tsx

```diff
@@ -27,12 +27,12 @@ export const Tuple = ({ abiTupleParameter, setParentForm, parentStateObjectKey }
 
   return (
     <div>
-      <div className="collapse collapse-arrow">
+      <div className="collapse collapse-arrow bg-base-200 pl-2 py-1.5 mt-1 border-2 border-secondary">
```

Comment:
Should we delete `mt-1` to have the same spacing between type text and border as regular inputs?

If feels too close after deleting we could increase `ContractInput` from `gap-1` to `gap-2`